Foundation/NSNumber.swift

```diff
@@ -32,158 +32,523 @@ internal let kCFNumberSInt128Type = CFNumberType(rawValue: 17)!
 internal let kCFNumberSInt128Type: CFNumberType = 17
 #endif
 
+extension Int8 : _ObjectTypeBridgeable {
```

Comment:
This looks like a faithful copy of the overlay, how much effort do you think will be required to make this shared tween the overlay and SCLF?

Reply:
The protocol is called `_ObjectiveCBridgeable` in the overlay - giving it the same name here would be a start. The files will never be identical because SCLF has the `class NSNumber` definition which isn't in the overlay. But I think we can get them closer.